products/zerto.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Zerto
+category: app
+tags: zerto
+iconSlug: # N/A currently
+permalink: /zerto
+releasePolicyLink: https://help.zerto.com/bundle/Lifecycle.Matrix.HTML/page/product_version_lifecycle_matrix_for_zerto.html
+activeSupportColumn: End of Bug Fixes
+releaseColumn: false
+releaseDateColumn: true
+eolColumn: End of Support Life (EOSL)
+
+-   releaseCycle: "10.0 Update 2"
+    releaseDate: 2023-11-28
+    support: true # Next GA date
+    eol: 2024-11-28
+    link: "https://help.zerto.com/bundle/RN.HTML.10.0_U2/page/what_s_new_in_zerto_10_0_update_2.html"
+
+-   releaseCycle: "10.0 Update 1"
```

Comment:
```suggestion
-   releaseCycle: "10.0_u1"
    releaseLabel "10.0 Update 1"
```

`releaseCycle` is used in API URLs, so it's better to have a `releaseCycle` without spaces and all lowercase. And this notation is also used on https://help.zerto.com/bundle/RN.HTML.10.0_U1/page/what_s_new_in_zerto_10_0_update_1.html, so would be better to have both IMO. Can you update the other releases too ?